app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -355,23 +359,23 @@ const MakeDecision = ({ data }) => {
           appearance="topaz"
         >
           {isBulkOperation
-            ? formatMessage(messages.applyTo, {
+            ? formatMessage(messages.applyToItems, {
                 itemsCount: itemData.length,
               })
             : formatMessage(messages.applyWithOptions)}
         </GhostButton>
       </>
     );
   };
-  const toggleAccordionTab = (tabId) => {
-    setAccordionTabsState({ ...accordionTabsState, [tabId]: !accordionTabsState[tabId] });
+  const toggleAccordionTab = (tabId, tabState, setTabState) => {
```

Comment:
This is now redundant. Once you have created your own hook to manage accordion state, you can use it independently inside of `OptionsStepForm`.